src/views/v2/month/calendar-body/day/calendar-events/calendar-event/tooltip/date/featured.php

```diff
@@ -23,8 +23,9 @@
 }
 ?>
 <em
-	class="tribe-events-calendar-month__calendar-event-tooltip-datetime-featured-icon tribe-common-svgicon tribe-common-svgicon--featured"
+	class="tribe-events-calendar-month__calendar-event-tooltip-datetime-featured-icon"
 	aria-label="<?php esc_attr_e( 'Featured', 'the-events-calendar' ); ?>"
 	title="<?php esc_attr_e( 'Featured', 'the-events-calendar' ); ?>"
 >
+	<?php $this->template('components/icons/featured', [ 'classes' => ['tribe-events-calendar-month__calendar-event-tooltip-datetime-featured-icon-svg'] ] ) ?>
```

Comment:
* :no_entry_sign: Inline PHP statement must end with a semicolon
* :no_entry_sign: Expected 1 spaces after opening parenthesis; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: Missing space after array opener.
* :no_entry_sign: Missing space after array opener.
* :no_entry_sign: Missing space before array closer.
* :no_entry_sign: Missing space before array closer.

<sub>hash: dace48e1dcb9be8abd3d3ba1473c9991</sub>